src/libsync/CMakeLists.txt

```diff
@@ -15,6 +15,14 @@ if ( APPLE )
     )
 endif()
 
+if ( WIN32 )
+    list(APPEND OS_SPECIFIC_LINK_LIBRARIES
+        ws2_32
+    )
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -D_FILE_OFFSET_BITS=64")
+    set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -D_FILE_OFFSET_BITS=64")
```

Comment:
Why is this required?

It would be nice also if everything that is related to zsync lib was together so we could later split it in its own file.

Reply:
Because of size_t usage and assumptions by zsync. See webarchive comments.

Reply:
Regarding putting all the changes together in one place, I can do that, no problem.

Reply:
Done.